docs/api_reference.rst

```diff
@@ -30,6 +30,7 @@ Distributions
 
    GenExtreme
    histogram_utils.histogram_approximation
+   timeseries.DiscreteMarkovChain
```

Comment:
If you add it to `distributions.__init__.py` `__all__` the short path should work (and it will be easier for users to find)
 
```suggestion
   DiscreteMarkovChain
```

Reply:
Done